composer.json

```diff
@@ -49,15 +50,15 @@
         "enqueue/enqueue": "^0.8.29",
         "friendsofphp/php-cs-fixer": "^2.11",
         "friendsofsymfony/jsrouting-bundle": "^2.1",
-        "knplabs/knp-menu-bundle": "^2.2",
-        "lcobucci/jwt": "^3.3.1",
+        "knplabs/knp-menu-bundle": "^3.1",
+        "lcobucci/jwt": "^4.0",
         "liip/functional-test-bundle": "^4.0.0",
-        "liip/test-fixtures-bundle": "^1.0.0",
+        "liip/test-fixtures-bundle": "^2.0.0",
         "paragonie/paseto": "^1.0",
         "phpbench/phpbench": "^1.0",
-        "phpstan/phpstan": "^0.11.1",
-        "phpstan/phpstan-phpunit": "^0.11.0",
-        "phpunit/phpunit": "^7.2",
+        "phpstan/phpstan": "^0.12.33",
+        "phpstan/phpstan-phpunit": "^0.12.13",
+        "phpunit/phpunit": "^8.0|^9.0",
```

Comment:
`^8.5|^9.5`. No reason to support lower versions which are no longer maintained :)